Formula/jupyterlab.rb

```diff
@@ -158,8 +158,8 @@ class Jupyterlab < Formula
   end
 
   resource "jsonschema" do
```

Comment:
Can this be replaced by the formula?

Reply:
What do you mean? You want it to `depends_on` on the formula? In which case do you use one or the other one?

Reply:
I want to try to depend on the jsonschema formula, yes.

Reply:
Pushed the change; a new long build will be required though.

Reply:
This won't work without some `.pth` / PYTHONPATH manipulation since `jsonschema` is installed inside a virtualenv.

Reply:
Then I've no idea about how this should be done.